tests/cluster/delta/test_filehandle.py

```diff
@@ -151,6 +166,30 @@ def test_02_read_avro(self):
         o.execute()
         self.assertEqual(6, result.count())
 
+    def test_03_read_avro_batch(self):
```

Comment:
What about also a json test?
So we test that it can load another type?

Reply:
but.. maybe 1 test is good enough... I mean.. the amount of tests that should be made for each file-type could increse.